eureka-webapp/src/main/webapp/eureka/module.js

```diff
@@ -71,13 +71,21 @@
        });  
     }
 
-    function eurekaConfig($stateProvider, $urlRouterProvider){
+    function eurekaConfig($stateProvider, $urlRouterProvider, $httpProvider){
 
+        if (!$httpProvider.defaults.headers.get) {
+            $httpProvider.defaults.headers.get = {};    
+        }    
+        $httpProvider.defaults.headers.get['If-Modified-Since'] = 'Mon, 26 Jul 1997 05:00:00 GMT';
+        $httpProvider.defaults.headers.get['Cache-Control'] = 'no-cache, no-store, must-revalidate';
+        $httpProvider.defaults.headers.get['Pragma'] = 'no-cache';
+        
         $urlRouterProvider.otherwise('/index');
 
         $stateProvider
             .state('index', {
                 url: '/index',
+                cache: false,
```

Comment:
Is this for caching the contents within angular for when a user goes back to the index page? If so, take that out, and I'll commit the other part. I was just looking for eliminating the caching that occurs when the user first navigates to Eureka.